lib/certificates.js

```diff
@@ -201,13 +202,8 @@ async function create(pg_pool, req, res, regex) {
   } else {
     payload.request = uuid.v4()
   }
-  let item_created = await insert_query(pg_pool, request_to_db(payload))
-  payload.domain_names.forEach((domain_name) => {
-    insert_names_query(pg_pool, [uuid.v4(), payload.id, domain_name, domain_name === payload.common_name]).catch((err) => {
-      console.error('ERROR: An error occurred inserting the domain name:', domain_name, 'on payload', payload);
-      console.error('ERROR: This will require manually inserting the domain name for the above record.');
-    });
-  });
+  await insert_query(pg_pool, request_to_db(payload))
+  await Promise.all(payload.domain_names.map( async (domain_name) => insert_names_query(pg_pool, [uuid.v4(), payload.id, domain_name, domain_name === payload.common_name])));
   payload.region = {
```

Comment:
certificate tests were failing, i found that the test to check to not allow duplicate certificates would fail becuase this used to asyncronously insert the record and on occasion the second certificate request would come in before postgresql finished inserting the first record.  Now it waits for the insert to complete before returning making the certificate tests more reliable.